### PR TITLE
feat(CDN): add new sdk to query cdn domain detail

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:

add new sdk to query cdn domain detail

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

- The sdk `Get` is Deprecated.

**Release note**:

```release-note
```

### CODE REVIEW

File: openstack/cdn/v1/domains/results.go [hunk 1]
```diff
@@ -59,6 +59,45 @@ type CdnDomain struct {
...
+	ID string `json:"id"`
+	// the acceleration domain name
+	DomainName string `json:"domain_name"`
+	// the service type, valid values are web, download, video and wholeSite
+	BusinessType string `json:"business_type"`
+	// the status of the acceleration domain name.
+	DomainStatus string `json:"domain_status"`
+	// the CNAME of the acceleration domain name
+	CName string `json:"cname"`
+	// the domain name or the IP address of the origin server
```
Comment (jinyangyang222) [inline openstack/cdn/v1/domains/results.go:85]: Please verify the description information

File: openstack/cdn/v1/domains/results.go [hunk 2]
```diff
@@ -59,6 +59,45 @@ type CdnDomain struct {
 	ModifyTime time.Time `json:"-"`
 }
 
+type DomainSourcesDetail struct {
```
Comment (jinyangyang222) [inline openstack/cdn/v1/domains/results.go:62]: add annotations for structure and parameters

File: openstack/cdn/v1/domains/results.go (hunk 1 above)
Comment (deer-hang) [reply openstack/cdn/v1/domains/results.go]: OK

File: openstack/cdn/v1/domains/results.go (hunk 2 above)
Comment (deer-hang) [reply openstack/cdn/v1/domains/results.go:62]: I prefer to keep the status quo.

**Pull request merged**